data/typelibrary/iec61131-3-1.0.0/typelib/bitwiseOperators/OR_16.fbt

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="OR_16" Comment="FB to calculate bitwise boolean OR (generic FB)">
+	<Identification Standard="61131-3" Classification="standard bitwise boolean function" Description="Copyright (c) 2014 Profactor GmbH &#10;  &#10;This program and the accompanying materials are made &#10;available under the terms of the Eclipse Public License 2.0 &#10;which is available at https://www.eclipse.org/legal/epl-2.0/ &#10; &#10;SPDX-License-Identifier: EPL-2.0" >
```

Comment:
I think I was not clear enough the (c) line should be:

Copyright (c)  2014, 2024 Profactor GmbH, HR Agrartechnik GmbH

As you took a file created by profactor and modified it. So both you and profactor have a copyright on that new file.